src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -9586,7 +9586,7 @@ private bool UriEqual(Uri uri1, string uri1Str, string uri2Str, XmlResolver reso
 
         /// <summary>
         /// This method should be called every time the reader is about to consume some number of
-        ///   characters from the input. It will count it agains the security counters and
+        ///   characters from the input. It will count it against the security counters and
         ///   may throw if some of the security limits are exceeded.
```

Comment:
@stephentoub Thanks! Corrected.
> It will count it against the security counters and may throw if some of the security limits are exceeded.

I'll tag the ones I can't decipher in the future more clearly.